public/js/controllers/groupsSection.controller.js

```diff
@@ -10,5 +11,104 @@
         /* jshint validthis: true */
         var vm = this;
 
+        vm.viewGroup = viewGroup;
+        vm.currentGroup = {};
+
+        vm.searchForGroups = searchForGroups;
+        vm.searchString = "";
+        vm.searchResults = [];
+
+        vm.isInGroup = isInGroup;
+        vm.joinGroup = joinGroup;
+        vm.startEditGroup = startEditGroup;
+        vm.leaveGroup = leaveGroup;
+
+        vm.inviteUser = inviteUser;
+        vm.invitationText = "";
+        vm.currentGroupMembers = [];
+        vm.currentGroupNonMembers = [];
+        vm.getGroupMembers = getGroupMembers;
+
+        activate();
+
+        function activate() {
+            vm.searchResults = [
+                {
+                    id: "5",
+                    name: "Test group",
+                    description: "This is just a test group; nothing more, nothing less.",
+                    users: ["stephen-lt-sl", "Melamoto"]
+                }, {
+                    id: "6",
+                    name: "Test group 2",
+                    description: "This is another test group; it might be a little less.",
+                    users: ["Melamoto"]
+                }
+            ];
+            viewGroup(vm.searchResults[0]);
+        }
+
+        function viewGroup(group) {
+            vm.currentGroup = group;
+            setCurrentGroupMemberships();
+        }
+
+        function searchForGroups() {
+            console.log("Searching for " + vm.searchString);
```

Comment:
I assume these are all stub functions? Although right now it's obvious, its good to explicitly say somewhere that part of the check in is stubbed, in case it is less obvious in a different case.